presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorPageSinkProvider.java

```diff
@@ -59,7 +59,7 @@ public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHa
 
         RaptorOutputTableHandle handle = (RaptorOutputTableHandle) tableHandle;
         return new RaptorPageSink(
-                new FileSystemContext(session),
+                new HdfsContext(session.getIdentity()),
```

Comment:
We also need all other information other than identity. Schema and table are available in `handle`. Same for all other places.

Reply:
Got it. I see that I can get schema and table names from RaptorOutputTableHandle:
```suggestion
                new HdfsContext(session, handle.getSchemaName(), handle.getTableName())
```